graphene_django/tests/test_converter.py

```diff
@@ -196,6 +196,24 @@ class Meta:
     convert_django_field_with_choices(field)
 
 
+def test_field_with_choices_underscore():
+    field = models.CharField(
+        choices=(
+            ("__amount__", "Amount"),
+            ("__percentage__", "Percentage"),
+        ),
+    )
+
+    class UnderscoreChoicesModel(models.Model):
+        ourfield = field
+
+        class Meta:
+            app_label = "test"
+
+    graphene_type = convert_django_field_with_choices(field)
+    assert len(graphene_type._meta.enum.__members__) == 2
```

Comment:
Can you assert the actual enum values here as well? So that it's clear what you expect to happen.

Reply:
Done.